### ISSUE TITLE
Please enable the wiki on this repo

### ISSUE THREAD

**Issue by ugermann** [uoe-eidf-cluster-users/eidf-epcc-cluster#32]
Title says it all: Please enable the wiki on this repo, so that we can add documentation easily.

**Comment by AntreasAntoniou** [comment]
I have admin powers, but will tag @pminervini who was the original creator so he is aware that I am going to make this change.

Oh wait, the wiki seems enabled?

**Comment by ugermann** [comment]
I don't think it is, or not accessible to everyone. I should be able to see a tab in the line that says "Code Issues Pull Requests ..." but I don't. Please check.

**Comment by AntreasAntoniou** [comment]
I can see it on my end. I'll have to look to see what's going on.

**Comment by AntreasAntoniou** [comment]
<img width="769" alt="image" src="https://github.com/uoe-eidf-cluster-users/eidf-epcc-cluster/assets/10792502/eb9fa58b-92d6-4ef2-b99f-d07070524a2f">

I just made you an admin. Maybe the wiki is only visible to collaborators? I am not sure what's going on.
